docs/shared/react-standalone-tutorial/react-standalone.md

```diff
@@ -0,0 +1,799 @@
+
+## Creating a new React App
+
+Create a new standalone React application with the following command:
+
+```shell {% command="npx create-nx-workspace@latest myreactapp --preset=react-standalone" path="~" %}
+? Bundler to be used to build the application …
+Vite    [ https://vitejs.dev/ ]
+Webpack [ https://webpack.js.org/ ]
+Rspack  [ https://www.rspack.dev/ ]
+```
+
+You can choose any bundler you like. In this tutorial we're going to use Vite. The above command generates the following structure:
+
+```
+└─ myreactapp
+   ├─ ...
+   ├─ e2e
+   │  └─ ...
+   ├─ public
+   │  └─ ...
+   ├─ src
+   │  ├─ app
+   │  │  ├─ app.module.css
+   │  │  ├─ app.spec.tsx
+   │  │  ├─ app.tsx
+   │  │  └─ nx-welcome.tsx
+   │  ├─ assets
+   │  ├─ main.tsx
+   │  └─ styles.css
+   ├─ index.html
+   ├─ nx.json
+   ├─ package.json
+   ├─ project.json
+   ├─ tsconfig.app.json
+   ├─ tsconfig.json
+   ├─ tsconfig.spec.json
+   └─ vite.config.ts
+```
+
+The setup includes..
+
+- a new React application at the root of the Nx workspace (`src/app`)
+- a Cypress based set of e2e tests (`e2e/`)
+- Prettier preconfigured
+- ESLint preconfigured
+- Jest preconfigured
+
+Let me explain a couple of things that might be new to you.
+
+| File           | Description                                                                                                                                                                                                                                          |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `nx.json`      | This is where we fine-tune how Nx works and define what [cacheable operations](/core-features/cache-task-results) there are, our [task pipeline](/concepts/task-pipeline-configuration). More on that soon.                                          |
+| `project.json` | This file contains the target that can be invoked for the `myreactapp` project. It is like a more evolved version of simple `package.json` scripts with more metadata attached. You can read more about it [here](/reference/project-configuration). |
+
+## Serving the App
+
+The most common tasks are already mapped in the `package.json` file:
+
+```json {% fileName="package.json" %}
+{
+  "name": "reactutorial",
+  "scripts": {
+    "start": "nx serve",
+    "build": "nx build",
+    "test": "nx test"
+  }
+  ...
+}
+```
+
+To serve your new React application, just run: `npm start`. Alternatively you can directly use Nx by using
+
+```shell
+nx serve
+```
+
+Your application should be served at [http://localhost:4200](http://localhost:4200).
+
+Nx uses the following syntax to run tasks:
+
+![Syntax for Running Tasks in Nx](/shared/react-standalone-tutorial/run-target-syntax.svg)
```

Comment:
this image doesn't seem to be loading. Looks like it could be a bad path or wasn't added?
![WkMacM1-3GW5Q6pP_06210826](https://github.com/nrwl/nx/assets/23272162/dca1b10d-7f1e-483e-8b9b-67f130485b25)

Reply:
oops, good catch. Fixed